files/anitya_cron.py

```diff
@@ -1,19 +1,29 @@
 #!/usr/bin/env python
 #-*- coding: utf-8 -*-
 
+import sys
+import logging
+import itertools
+import multiprocessing
+
 import anitya
 import anitya.app
 import anitya.lib.exceptions
 import anitya.lib.model
 
-import logging
-import sys
 
-PBAR = True
-try:
-    from progressbar import Bar, ETA, Percentage, ProgressBar, RotatingMarker
-except ImportError:
-    PBAR = False
+def update_project(project):
+    """ Check for updates on the specified project. """
+    print(project.name)
```

Comment:
it seems that ``print()`` works fine in python 2.7 without the import of ``__future__`` (tested on pkgdb02 with python 2.7.5).

That being said, it should be a call to the logger anyway :)